### ISSUE TITLE
Typos on website

### ISSUE THREAD

**Issue by Neustradamus** [Spider-Admin/Frost#1]
Dear @Spider-Admin,

Thanks for your work since a long time.

I have seen two mistakes and it has been done two times (copy/paste) on:
- https://jtcfrost.sourceforge.net/

-> Please see donwload page for more infomration how to obtain it

donwload
infomration

Can you solve historical mistakes?

Thanks in advance.

**Comment by Spider-Admin** [comment]
Thanks for the hint. I have fixed the spelling mistakes.

What do you mean by “Can you solve historical mistakes?”

**Comment by Neustradamus** [comment]
@Spider-Admin: Thanks for this quick answer and changes.
It was very old bad words, more 10 years -> historical ^^

**Comment by Neustradamus** [comment]
@Spider-Admin: Can you update the SF project at the same time?

Can you add a "move" box like https://sourceforge.net/projects/handbrake for example.
- https://sourceforge.net/p/jtcfrost/

Can you add the new place in description of the project?
-> Hyphanet + GitHub link

Can you remove old parts:
- https://sourceforge.net/p/jtcfrost/code/
- https://sourceforge.net/p/jtcfrost/_list/git
- https://sourceforge.net/p/jtcfrost/discussion/
- https://sourceforge.net/p/jtcfrost/news/
- https://sourceforge.net/p/jtcfrost/_list/tickets
- https://sourceforge.net/p/jtcfrost/mailman/

Can you add on website, link/button to GitHub?
- https://jtcfrost.sourceforge.net/

To disable/delete tabs, go here: https://sourceforge.net/projects/jtcfrost/
To add a new tab, go here: https://sourceforge.net/projects/jtcfrost/
To change informations, go here: https://sourceforge.net/p/jtcfrost/admin/overview
To export, go here: https://sourceforge.net/p/jtcfrost/admin/export

The new place can be added in:
- "Moved Project to", here: https://sourceforge.net/p/jtcfrost/admin/overview
- A new tab, go here: https://sourceforge.net/projects/jtcfrost/

Thanks in advance.

**Comment by Spider-Admin** [comment]
Yes, I can change the entire project at SourceForge, because I have been promoted to admin.

https://github.com/Spider-Admin/Frost/ is my private repository, the official repository is at SourceForge. Therefore I will not link to my private repository.

I have renamed the old SVN to make it clearer that Git is now being used, but I will not link to my private repository.

The other points are aiming to replace SourceForge with GitHub. I also have this on my wish list, but it won't happen today. I would also let the community decide here.

My goal would be the following:
- Source code on GitHub under a URL like https://github.com/freenet-frost/frost/
- Tickets on GitHub
- Releases on Github
- Source code and releases are transferred to SourceForge as automatically as possible.
- Website at GitHub or SourceForge?

**Comment by Spider-Admin** [comment]
"frost" as name for a user/organization is already used: https://github.com/frost/ So we need another name:
- jtcfrost (JTC = **J**an-**T**homas **C**zornack) is outdated, because Jan is no longer a developer.
- frost-project?
- freenet-frost?
- hyphanet-frost?

I would let the community vote for this too.

**Comment by Neustradamus** [comment]
@Spider-Admin: Thanks for your answers!

If you will arrive to have https://github.com/freenet-frost/frost/, please do correctly with a transfer, not a fork.

If you create a free GitHub organization: https://github.com/account/organizations/new
-> frost-project

You can transfer easily all repositories linked to frost into without lost, GitHub adds redirections.

For Spider, you can create another one too easily.

After, you can rename your personal account too ^^

**Comment by Spider-Admin** [comment]
I created a organization named "frost-project" and moved all relevant repositories to this organization. I also invited all old known members.
